conf/audit_sql.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this file used in any way to build the Pentaho container image? If not, please remove it from git.